onboarding/src/Components/QA/QAPage.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import Navbar from "../Navbar";
+import LeftMenu from "../LeftMenu";
+import PageAddressBar from "../PageAddressBar";
+import QAList from "./QAList";
+
+const QAPage = () => {
```

Comment:
This boring `loggedUser` again :D :/
`const QAPage = (props) {
let loggedUser = props.location.state.loggedUser;`
to keep him all the time.